package.json

```diff
@@ -14,7 +14,10 @@
     "build": "yarn run generate_licenses && yarn run clean:dist && node --max_old_space_size=8192 node_modules/@angular/cli/bin/ng build",
     "build:prod": "node scripts/setup_prod.js && yarn run build -- --prod",
     "build:prod:aot": "yarn run build:prod --source-map --base-href /ui/ && node scripts/verify_build.js",
-    "test": "ng test",
+    "jest": "./node_modules/.bin/jest",
+    "test": "jest",
```

Comment:
Doesn't it work without having 'jest' command?

Reply:
ng and jest require global installation for those commands to work. I added those aliases for convenience.